src/blockdata/script.rs

```diff
@@ -225,9 +225,9 @@ impl From<bitcoinconsensus::Error> for Error {
     }
 }
 
-/// Helper to encode an integer in script format.
+/// Helper to encode an integer in script(minimal CScriptNum) format.
```

Comment:
Can I ask a question, why is the open parenthesis used with no space before it in docs? I've seen this elsewhere and wondered why its done like that, this is the first PR I've seen add one - hence my question. Cheers

Reply:
I had no idea that we are supposed to put spaces there. Won't be dismissing the two ACKs for this nit

Reply:
There should be a space there -- but agreed, we can fix this in a later doc-fixup PR.

Reply:
No need to fix it, I can do them when I'm cleaning up docs. I just wanted to make sure it wasn't for a reason I did not know about. Cheers